### ISSUE TITLE
Manually add an account on GNU/Linux

### ISSUE THREAD

**Issue by CyrilWaechter** [specklesystems/speckle-blender#25]
From [docs](https://speckle.guide/user/manager.html#adding-accounts-manually) I see that there is apparently no SpeckleManager for GNU/Linux systems. Although I see also that it is possible to [add an account manually](https://speckle.guide/user/manager.html#adding-accounts-manually). 

Can you please point me out where in the code it looks for this .json file to see where I need to put it or if code needs to be modified ?

**Comment by didimitrie** [comment]
Good catch! It seems like the py sdk does not have this yet (any of that code should most likely be [here](https://github.com/specklesystems/speckle-py/blob/5a9b4137da5558d8f460e305401ff9417d8679a5/specklepy/api/credentials.py). @izzylys's away today, but she'll get on top of this tomorrow! 

If you're curious, the .net [implementation is here](https://github.com/specklesystems/speckle-sharp/blob/master/Core/Core/Credentials/AccountManager.cs#L134).
